### DISCUSSION TITLE
Xamarin + Android + Plugin.LocalNotification 10.1.8

### DISCUSSION THREAD

**Discussion by ckoziel** [thudugala/Plugin.LocalNotification#481] [Q&A] [answered]
I have a quick question. On Android, if Notifications are not enabled on the device, does Plugin.LocalNotifcation still queue the notification? Or do you need the Notification Permission on the device to be Allowed before you can schedule a notification?

**Comment by thudugala** [comment, marked as answer]
@ckoziel Yes, from 10.1.8 onwards, if notifications are not allowed, you cannot schedule notifications

**Comment by ckoziel** [comment]
Thank you Elvin.


Warm Regards,

Chris Koziel

President

[1479332777487_200.png]

Email: ***@***.******@***.***>

Phone: 630.461.8778

Website: www.visualwebsite.com<http://www.visualwebsite.com>

Lemont, IL 60439


Visualwebsite.com has been featured in Voyage Chicago Magazine. Click here to read the article.<http://voyagechicago.com/interview/meet-chris-koziel-visualwebsite-com-lemont/>





________________________________
